internal/authtoken/repository.go

```diff
@@ -149,15 +153,15 @@ func (r *Repository) LookupAuthToken(ctx context.Context, id string, opt ...Opti
 		if errors.IsNotFoundError(err) {
 			return nil, nil
 		}
-		return nil, fmt.Errorf("auth token: lookup: %w", err)
+		return nil, errors.Wrap(err, op)
 	}
 	if opts.withTokenValue {
 		databaseWrapper, err := r.kms.GetWrapper(ctx, at.GetScopeId(), kms.KeyPurposeDatabase, kms.WithKeyId(at.GetKeyId()))
 		if err != nil {
-			return nil, fmt.Errorf("lookup: unable to get database wrapper: %w", err)
+			return nil, errors.Wrap(err, op, errors.WithCode(errors.Encrypt))
```

Comment:
`r.kms.GetWrapper` should already be returning a valid domain error with code, should not need to wrap with code